### PR TITLE
refactor: Do not repeat `dataDir`

### LINKED ISSUE #200: Refactor(DRY): `./data/${name}`
Easy to break this convention in new PRs

image

e.g.:  https://github.com/juspay/services-flake/pull/195#discussion_r1615217405

### PR DESCRIPTION
**Pull request by shivaraj-bh**
resolves #200

### CODE REVIEW

File: nix/lib.nix [hunk 1]
```diff
@@ -15,6 +15,11 @@
       ];
       serviceModule = { config, name, ... }: {
         options = {
+          dataDir = lib.mkOption {
+            type = lib.types.str;
+            default = "./data/${name}";
+            description = "The ${service}.<name> data directory";
```
Comment (shivaraj-bh) [inline nix/lib.nix:21]: @srid How does this look for a description? In the docs the option will be rendered as `<name>` anyways?

context: https://github.com/juspay/services-flake/pull/258#discussion_r1666999402

**Comment by shivaraj-bh** [comment]
I also think we can refactor `enable` option in the similar fashion

File: nix/lib.nix (hunk 1 above)
Comment (srid) [reply nix/lib.nix:21]: Sure (but hard to say definitely without rendering the docs though)

Comment (srid) [inline nix/lib.nix:21]:
```suggestion
            description = "The directory where all data for `${service}.<name>` is stored";
```

**Review by srid — APPROVED** [review]

**Pull request merged**